docs/api.md

```diff
@@ -1,109 +1,103 @@
 ---
 title: API reference
-description: Endpoints for the Wave API
-hide:
-    - navigation
-    - footer
+sidebar_position: 12
 ---
 
-# API reference
+This page summarizes the API provided by the Wave container service.
 
-This page summarizes the API provided by the Wave service.
+## `/container-token`
```

Comment:
shouldn't we provide the HTTP method for this endpoint?
in this case its POST

Reply:
@munishchouhan, I can create a new PR specifically to update the API methods.